pallets/hello-substrate/src/lib.rs

```diff
@@ -19,8 +19,8 @@ decl_module! {
 			// Ensure that the caller is a regular keypair account
 			let caller = ensure_signed(origin)?;
 
-			// Print a message
-			print("Hello World");
+			// Logs a message
+			debug::info!("Hello World");
```

Comment:
That requires that the `RuntimeLogger` initialized.  For `print` you just need `-lruntime=debug`.

Reply:
Here are some docs about the RuntimeLogger https://substrate.dev/rustdocs/master/frame_support/debug/struct.RuntimeLogger.html

It seems `debug::info!` messages are printed in this case because runtime is native.

@bkchr Is there a reliable way to just print a message to the terminal that will reliably print in native or wasm without custom flag?

Reply:
Not really. Can we not document that you need a flag to turn on runtime printing?

Reply:
Yes we can. And we will.

I was just trying to be as beginner-friendly as possible for this introductory context.

We can still easily proceed.